src/pawns.cpp

```diff
@@ -172,6 +175,13 @@ namespace {
             backward = (b | shift_bb<Up>(b)) & theirPawns;
         }
 
+        if (!backward && relative_rank(Us, s) == RANK_5)
+        {
+            if (pawn_attack_span(Us, s + pawn_push(Us)) & theirPawns)
+                score += EnPassantBonus;
+
+        }
+
```

Comment:
Why not simply

+ if (     !backward
       && relative_rank(Us, s) == RANK_5
       && pawn_attack_span(Us, s + pawn_push(Us)) & theirPawns)
       score += EnPassantBonus;